src/Rules/Deprecations/EchoDeprecatedToStringRule.php

```diff
@@ -0,0 +1,115 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Deprecations;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Echo_;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\RuleLevelHelper;
+use PHPStan\Type\ErrorType;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\Type;
+
+/**
+ * @implements \PHPStan\Rules\Rule<Echo_>
+ */
+class EchoDeprecatedToStringRule implements \PHPStan\Rules\Rule
```

Comment:
The rule isn't registered in rules.neon. Because of a minor version, it should be done like here: https://github.com/phpstan/phpstan-strict-rules/blob/4efb461f334c92b77ca8d632c3b29cfa3cb4b810/rules.neon#L42-L44